discovery-provider/src/queries/get_random_tracks.py

```diff
@@ -0,0 +1,32 @@
+import logging  # pylint: disable=C0302
+import random
+
+from src.api.v1.helpers import extend_track, decode_string_id
+from src.queries.get_trending_tracks import get_trending_tracks
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_RANDOM_LIMIT = 10
+
+def get_random_tracks(args):
+    """Gets random tracks from trending by getting the currently cached tracks and then populating them."""
+    exclusion_list = args.get("exclusion_list") or []
+    time = args.get("time") if args.get("time") is not None else 'week'
```

Comment:
nit: I think this can just be `args.get("time", "week")`, since the second argument to get is the default val if the key doesn't exist